ui/apps/dashboard/src/components/Insights/InsightsTabManager/InsightsHelperPanel/features/SchemaExplorer/SchemasContext/commonSchemas.ts

```diff
@@ -0,0 +1,13 @@
+import type { JSONSchema } from '@inngest/components/SchemaViewer/types';
+
+export const EVENT_SCHEMA_JSON: JSONSchema = {
+  title: 'event',
+  type: 'object',
+  properties: {
+    data: { type: 'string' },
```

Comment:
why is this type 'string'? it doesn't seem to affect this though 
<img width="251" height="178" alt="image" src="https://github.com/user-attachments/assets/a665ddca-9749-4c65-811d-fc4772a75801" />